scripts/check-mcp-compliance.py

```
@@ -0,0 +1,308 @@
+
+                try:
+                    content = py_file.read_text()
+                    for pattern in mock_patterns:
+                        if pattern in content:
+                            # Count occurrences
+                            count = content.count(pattern)
+                            issues.append(
+                                f"{py_file}: {count} occurrences of '{pattern}'"
+                            )
+                except Exception:
+                    pass
```

Ruff (0.11.9)

`try`-`except`-`pass` detected, consider logging the exception

(S110)